htlc/scripts/deploy.js

```diff
@@ -1,64 +1,66 @@
+		throw new Error(`Account ${name} is not defined`);
+	}
+	return account;
 }
 
 async function run(runtimeEnv, deployer) {
-  const masterAccount = await getDeployerAccount(deployer, 'master-account');
-  const bob = await getDeployerAccount(deployer, 'bob');
-  const alice = await getDeployerAccount(deployer, 'alice');
+	const masterAccount = await getDeployerAccount(deployer, "master-account");
+	const bob = await getDeployerAccount(deployer, "bob");
+	const alice = await getDeployerAccount(deployer, "alice");
 
-  const secret = 'hero wisdom green split loop element vote belt';
-  const secretHash = Buffer.from(sha256.digest(secret)).toString('base64');
+	const secret = "hero wisdom green split loop element vote belt";
+	const secretHash = Buffer.from(sha256.digest(secret)).toString("base64");
 
-  const scTmplParams = {
-    bob: bob.addr,
-    alice: alice.addr,
-    hash_image: secretHash,
-  };
+	const scTmplParams = {
+		bob: bob.addr,
+		alice: alice.addr,
+		hash_image: secretHash,
+	};
 
-  /** ** firstly we fund Alice and Bob accounts ****/
-  const bobFunding = {
-    type: types.TransactionType.TransferAlgo,
-    sign: types.SignType.SecretKey,
-    fromAccount: masterAccount,
-    toAccountAddr: bob.addr,
-    amountMicroAlgos: 10e6, // 10 Algos
-    payFlags: { note: 'funding account' },
-  };
-  // We need to copy, because the executeTransaction is async
-  const aliceFunding = Object.assign({}, bobFunding);
-  aliceFunding.toAccountAddr = alice.addr;
-  aliceFunding.amountMicroAlgos = 0.1e6; // 0.1 Algo
-  await Promise.all([
-    executeTransaction(deployer, bobFunding),
-    executeTransaction(deployer, aliceFunding),
-  ]);
+	/** ** firstly we fund Alice and Bob accounts ****/
+	const bobFunding = {
+		type: types.TransactionType.TransferAlgo,
+		sign: types.SignType.SecretKey,
+		fromAccount: masterAccount,
+		toAccountAddr: bob.addr,
+		amountMicroAlgos: 10e6, // 10 Algos
+		payFlags: { note: "funding account" },
+	};
+	// We need to copy, because the executeTransaction is async
+	const aliceFunding = Object.assign({}, bobFunding);
+	aliceFunding.toAccountAddr = alice.addr;
+	aliceFunding.amountMicroAlgos = 0.1e6; // 0.1 Algo
+	await Promise.all([
+		deployer.executeTx(bobFunding),
+		deployer.executeTx(aliceFunding),
+	]);
 
-  /** ** now bob creates and deploys the escrow account ****/
-  console.log('hash of the secret:', scTmplParams.hash_image);
-  // hash: QzYhq9JlYbn2QdOMrhyxVlNtNjeyvyJc/I8d8VAGfGc=
+	/** ** now bob creates and deploys the escrow account ****/
+	console.log("hash of the secret:", scTmplParams.hash_image);
+	// hash: QzYhq9JlYbn2QdOMrhyxVlNtNjeyvyJc/I8d8VAGfGc=
 
-  await deployer.fundLsig(
-    'htlc.py',
-    { funder: bob, fundingMicroAlgo: 2e6 },
-    {},
-    [],
-    scTmplParams
-  );
+	await deployer.mkContractLsig("HTLC_Lsig", "htlc.py", scTmplParams);
 
-  // Add user checkpoint
-  deployer.addCheckpointKV('User Checkpoint', 'Fund Contract Account');
+	await deployer.fundLsig(
```

Comment:
You missed `scTmplParams` paramter. Pls check

Reply:
I don't think this requires that https://github.com/scale-it/algo-builder/blob/master/packages/algob/src/types.ts#L435